### ISSUE TITLE
use of H0 tags which the W3C validator complains about

### ISSUE THREAD

**Issue by glenux** [dr4Ke/editsections#5]
Christophe Drevet said "it's a dirty trick to display the edit button for the first section". Maybe the same result  could be achieved using standard CSS selectors ? 

Like  :

<pre>
#container h1:first-child { ...  }
</pre>

Just try this on your own wiki: http://validator.w3.org/

**Comment by dr4Ke** [comment]
Yes, probably, but I don't know much about that. Could you make a patch using this solution ?
